daemon/manager/manager.go

```diff
@@ -101,17 +101,23 @@ func Execute(configs *options.BootstrapConfigs) error {
 	log.Info("command Start cni successful")
 	defer cniInstaller.Stop()
 
-	setupCloseHandler()
+	*configs.Status = setupCloseHandler()
```

Comment:
Assign an int to a pointer?

Reply:
Because I want the data in configs.Status and configs.BpfConfig.Status to be the same at the same time

Reply:
I add a KmeshStartStatus in bpf